src/main/scala/io/dronekit/CouchbaseStreamsWrapper.scala

```diff
@@ -292,6 +295,24 @@ class CouchbaseStreamsWrapper(host: String, bucketName: String, password: String
     observableToSource(docObservable).map(json => convertToEntity[T](json))
   }
 
+  /**
+   * Does a batch update of keys -> each update
+   * @param entities Seqence of BatchUpdateRequest to update
```

Comment:
Looks like you could change the type to UpdateObject. It's confusing to have a type named BatchUpdateEntity when it's really a single update entity.